src/ripple/app/main/NodeStoreScheduler.cpp

```diff
@@ -47,11 +47,14 @@ void NodeStoreScheduler::onChildrenStopped ()
 
 void NodeStoreScheduler::scheduleTask (NodeStore::Task& task)
 {
-    ++m_taskCount;
-    m_jobQueue->addJob (
-        jtWRITE,
-        "NodeObject::store",
-        [this, &task] (Job&) { doTask(task); });
+    if (! isStopping())
+    {
+        ++m_taskCount;
+        m_jobQueue->addJob (
+            jtWRITE,
+            "NodeObject::store",
+            [this, &task] (Job&) { doTask(task); });
+    }
```

Comment:
Yup, I hear you.  These were minimal changes that removed crashes I could verify actually occur and could reproduce.  To truly tighten up the app to be watertight would be a much bigger challenge.

I modeled these three changes after the pre-existing fix on line 63, where `isStopping()` was already being used to plug a leak.  Not perfect, but a step in the right direction, and guaranteed to significantly reduce crashes on shutdown (at least on my two Macs :smile:).